iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -496,5 +515,24 @@ private static double GenerateTemperatureWithinRange(int max = 50, int min = 0)
         {
             return Math.Round(s_random.NextDouble() * (max - min) + min, 1);
         }
+
+        private async Task UpdateForEmptyWritableProperty(string componentName, CancellationToken cancellationToken)

```

Comment:
Maybe we should make this method more generic and not hard code the property name in the method, like passing the property name as a parameter?

Reply:
sure! Actually I am re-constructing this method, and will bring this to the new one.